src/test/java/service/JanngiServiceTest.java

```diff
@@ -0,0 +1,57 @@
+package service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import domain.Board;
+import domain.fake.FakeJanggiDao;
+import domain.fake.FakePieceDao;
+import domain.fake.InitialChessPiecePositionsWithOutKingGenerator;
+import domain.position.ChessPiecePositions;
+import domain.position.InitialChessPiecePositionsGenerator;
+import game.Janggi;
+import game.Turn;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class JanngiServiceTest {
+
+    @Test
+    @DisplayName("왕이 죽으면 게임이 종료 된다.")
+    void test1() {
+        //given
+        final ChessPiecePositions chessPiecePositions = ChessPiecePositions.from(new InitialChessPiecePositionsWithOutKingGenerator());
+        final Turn turn = Turn.create();
+        final Board board = new Board(chessPiecePositions);
+        final Janggi janggi = new Janggi(board, turn);
+        final FakePieceDao fakePieceDao = new FakePieceDao();
+        final FakeJanggiDao fakeJanggiDao = new FakeJanggiDao();
```

Comment:
상위 타입을 선언해주는게 좋을 것 같아요. 😉 

```suggestion
        final PieceDao fakePieceDao = new FakePieceDao();
        final JanggiDao fakeJanggiDao = new FakeJanggiDao();
```